### ISSUE TITLE
Enabling my webcam makes the qTox window become unresponsive

### ISSUE THREAD

**Issue by nikwen** [qTox/qTox#995]
I'm having issues with the camera on qTox. It works fine with uTox.

What I did:
- Open qTox
- Open the audio/video settings

What happened:
- The preview window showed one frame
- The qTox window became unresponsive (the OS asked me whether I wanted to enforce closing it)

The webcam doesn't work either when trying to do a video call.

System information:

I'm running Ubuntu 14.10 64 bit.
I'm using the following webcam: http://www.logitech.com/en-us/product/hd-webcam-c270

Log (removed my ID):

```
nikwen@nikwen:~$ qtox
Settings: Loading from  "/home/nikwen/.config/tox/qtox.ini" 
Using embeded bootstrap nodes list 
built on:  05:06:20 Jan  7 2015 ( 1420625129 ) 
commit:  built without git 

IPC: Our global ID is  <REMOVED VALUE>
IPC: Creating the global shared memory and taking ownership 
Loaded translation "de" 
Core: loading Tox from "/home/nikwen/.config/tox/<REMOVED VALUE>.tox" 
Audio:  "Trying to open output Internes Audio Analog Stereo" 
Audio: Opening audio output  "Internes Audio Analog Stereo" 
Audio:  "Trying to open input Webcam C270 Analog Mono" 
Audio: Opening audio input  "Webcam C270 Analog Mono" 
Core starting with IPv6 enabled 
Core::loadConfiguration: reading from  "/home/nikwen/.config/tox/<REMOVED VALUE>.tox" 
Core: Bootstraping to the DHT ... 
"Core: Bootstraping from Munrek, addr 195.154.119.113, port 33445" 
"Core: Bootstraping from nurupo, addr 192.210.149.121, port 33445" 
Core: Connected to DHT 
Settings: Saving in  "/home/nikwen/.config/tox/qtox.ini" 
Core: writing tox_save to  "/home/nikwen/.config/tox/<REMOVED VALUE>.tox" 
VideoSurface: Init 
Audio:  "Trying to open output Internes Audio Analog Stereo" 
AL lib: (WW) FreeContext: (0x17c1b30) Deleting 1 Source(s)
Audio: Opening audio output  "Internes Audio Analog Stereo" 
Audio:  "Trying to open input Webcam C270 Analog Mono" 
Audio: Opening audio input  "Webcam C270 Analog Mono" 
CameraWorker: Resume 
HIGHGUI ERROR: V4L2: Unable to get property Hue(9963779) - Das Argument ist ungültig
CameraWorker: Resolutions (QSize(160, 120) ,  QSize(320, 240) ,  QSize(432, 240) ,  QSize(640, 360) ,  QSize(640, 480) ,  QSize(800, 600) ,  QSize(960, 544) ,  QSize(960, 720) ,  QSize(1280, 720) ,  QSize(1280, 960) )  
VideoSurface: Resize pbo  3686400 ( QSize(1280, 960) ) bytes (before 0 )
```

The line causing the error seems to be the following one:
`HIGHGUI ERROR: V4L2: Unable to get property Hue(9963779) - Das Argument ist ungültig`

Note: "Das Argument ist ungültig" is German for "Invalid argument".

**Comment by nikwen** [comment]
I'm not sure anymore that the HIGHGUI error is really the one which is causing this behaviour.

I opened the system monitor in parallel and it showed that qTox is constantly using 100 % of one core when displaying the camera preview.
Therefore, if I wait long enough, I get to see that the camera image changes.

**Comment by nikwen** [comment]
Ok, please forget about this one. It's a performance issue, the window does not really freeze.
I'll open another bug report later. :)

**Comment by nikwen** [comment]
Sorry, I forgot to close this. I've since done some debugging and will open a new issue soon. ;)

**Comment by tux3** [comment]
> IPC: Our global ID is  <REMOVED VALUE>

Just a note, this is a completely random value, no need to expunge it :)

**Comment by nikwen** [comment]
Ah, ok. Thanks. :)
